zerver/openapi/zulip.yaml

```diff
@@ -6537,6 +6537,14 @@ paths:
                           A boolean which signifies whether the user has a zoom token and has thus completed
                           OAuth flow for the [Zoom integration](/help/start-a-call).  Clients need
                           to know whether initiating Zoom OAuth is required before creating a Zoom call.
+                      giphy_api_key:
+                        type: string
+                        description: |
+                          Present if `giphy` is present in `fetch_event_types`.
+
+                          Use this API key fetch GIFs from GIPHY. Please read
+                          https://support.giphy.com/hc/en-us/articles/360028134111-GIPHY-API-Terms-of-Service-
```

Comment:
We can make this a nicer markdown link rather than having the whole URL in the text.

Reply:
ok